From 18924bd5db4ea80fe8ff8eff7e20cf75036ff0c7 Mon Sep 17 00:00:00 2001
From: Thanh Quan <thanh.quan.xn@renesas.com>
Date: Wed, 21 May 2025 23:54:30 +0700
Subject: [PATCH 37/52] arm64: dts: renesas: r8a78000: Add Multi-Protocol PHY
 node

Support for the Multi-Protocol PHY controller on the R-Car X5H (R8A78000)
SoC by adding its device node to the device tree.

Signed-off-by: Thanh Quan <thanh.quan.xn@renesas.com>
Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 arch/arm/dts/r8a78000.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/dts/r8a78000.dtsi b/arch/arm/dts/r8a78000.dtsi
index 75c5714a3cb..b7e7ec860cb 100644
--- a/arch/arm/dts/r8a78000.dtsi
+++ b/arch/arm/dts/r8a78000.dtsi
@@ -377,6 +377,13 @@
 			interrupt-controller;
 		};
 
+		mp_phy: mp_phy@c9a00000 {
+			compatible = "renesas,multi-protocol-phy";
+			reg = <0 0xc9a00000 0 0x100000>;
+			#phy-cells = <2>;
+			status = "disabled";
+		};
+
 		sram: sram@c1000000 {
 			compatible = "arm,rcar-sram-ns", "mmio-sram";
 			reg = <0x0 0xc1000000 0x0 0x80000>;
-- 
2.34.1

